"use strict";
function promesaFallida() {
    return new Promise((_resolve, reject) => {
        console.log(`2 rechazando la promesa  тЪая╕П`);
        reject();
    });
}
console.log(`1 iniciando... тЪая╕П`);
promesaFallida()
    .then(() => { })
    .catch(() => console.log(`3 Atrapado ЁЯРЫ тЪая╕П`));
// 1 iniciando... тЪая╕П
// 2 rechazando la promesa  тЪая╕П
// 3 Atrapado ЁЯРЫ тЪая╕П
